File: spring-cloud-bus-demo/bus-common/src/main/java/com/livk/bus/event/LivkBusEvent.java
package com.livk.bus.event;

import org.springframework.cloud.bus.event.Destination;
import org.springframework.cloud.bus.event.RemoteApplicationEvent;

/**
 * <p>
 * LivkBusEvent
 * </p>
 *
 * @author livk
 * @date 2021/11/1
 */
public class LivkBusEvent extends RemoteApplicationEvent {

	public LivkBusEvent() {
		super();
	}

	public LivkBusEvent(Object source, String originService, Destination destination) {

		super(source, originService, destination);
	}

	@Deprecated
	public LivkBusEvent(Object source, String originService, String destinationService) {

		super(source, originService, destinationService);
	}

}

File: spring-cloud-bus/bus-commons/src/main/kotlin/com/livk/bus/event/LivkBusEvent.kt
package com.livk.bus.event

import org.springframework.cloud.bus.event.Destination
import org.springframework.cloud.bus.event.RemoteApplicationEvent

/**
 * @author livk
 */
class LivkBusEvent(source: Any, originService: String, destination: Destination) :
    RemoteApplicationEvent(source, originService, destination)
